train_management/templates/train_management/dayplanningtext_update_form.html

```diff
@@ -4,6 +4,12 @@
 
 {% block title %}{% translate "template.dayplanning_text.update.title" %}{% endblock %}
 
+{% block head %}
+
+  {{ form.media }}
```

Comment:
Was befindet sich in diesem Block? Um was handelt es sich bei `form.media` und wozu wird es gebraucht?

Reply:
https://docs.djangoproject.com/en/3.2/topics/forms/media/
Importiert die Tinymce-settings

Reply:
Hast du ausprobiert ob dies in Prod auch funktioniert?